platform/linux-generic/odp_schedule_scalable.c

```diff
@@ -927,6 +927,9 @@ static int _schedule(odp_queue_t *from, odp_event_t ev[], int num_evts)
 	}
 
 	cpu_id = odp_cpu_id();
+	if (odp_unlikely(cpu_id < 0))
```

Comment:
The cpu ID here should perhaps be thread ID, but that is orthogonal to this change, which looks ok.

Reply:
Yeah, I was thinking the same, but didn't want to touch the scalable scheduler code any more than necessary.